lib/sycamore/sycamore/utils/pdf.py

```diff
@@ -125,12 +125,18 @@ def convert_from_path_streamed_batched(filename: str, batch_size: int) -> Genera
         yield batch
 
 
-def pdf_to_image_files(pdf_path: str, file_dir: Path) -> Generator[Path, None, None]:
+def pdf_to_image_files(pdf_path: str, file_dir: Path, fh = None, pages = None) -> Generator[Path, None, None]:
```

Comment:
Do pdf_path: Union[str, io.IOBase]
That way callers only have to specify a single parameter rather than two related ones.